src/storage.ts

```diff
@@ -754,19 +1052,41 @@ export class Storage extends Service {
       }
     );
   }
+
+  /**
+   * Get a reference to an HmacKey object.
+   * Note: this does not fetch the HMAC key's metadata. Use HmacKey#get() to
+   * retrieve and populate the metadata.
+   *
+   * @param {string} accessId The HMAC key's access ID.
```

Comment:
@jkwlui could you add an optional parameter for projectId? This would help in the case the user doesn't want to use the project defined by credentials or GOOGLE_CLOUD_PROJECT.